db/db_compaction_test.cc

```diff
@@ -360,12 +361,33 @@ TEST_P(DBCompactionTestWithParam, CompactionDeletionTrigger) {
     for (int k = 0; k < kTestSize; ++k) {
       ASSERT_OK(Delete(Key(k)));
     }
-    ASSERT_OK(dbfull()->TEST_WaitForFlushMemTable());
+    Flush();
```

Comment:
I found it hard to reason about an unpredictable number of tombstones getting stuck in the memtable, so triggered a manual flush.

Reply:
Done.